fixieai/agents-ts/package.json

```diff
@@ -15,8 +15,11 @@
     "format": "dprint fmt",
     "format:check": "dprint check",
     "postinstall": "npm run build",
-    "build": "tsc -p .",
+    "build": "tsc -p . && chmod a+x dist/serve-bin.js",
```

Comment:
Certainly not something you need to focus on right now but we should try to support Windows assuming it's straightforward to do so. This bit me at the hackathon where like 99% of the code worked on Windows but a couple of critical pieces didn't for dumb reasons :)